### DISCUSSION TITLE
Sponsoring as organization/company

### DISCUSSION THREAD

**Discussion by Piedone** [community/community#22210] [Other Feature Feedback, Questions, & Ideas]
<p>We, a company, would like to sponsor other developers via GitHub Sponsors. While I can’t find any information on whether this is possible, or you can just sponsor as an individual, <a href="https://github.community/t/github-sponsors-faq/10217">the FAQ</a> does hint on it (“Will corporate sponsors receive an invoice?”). However, I see no option to do this. Is it possible?</p>
<p>If I go to the sponsor page of a developer I only see “Sponsor as [my name]”. There is no option to switch over to the organization that I’m an administrator of. I don’t see any option to enable this under the organization, should that be missing.</p>
<p>Thanks!</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/piedone">@piedone</a>,</p>
<p>Thank you for being here and for your interest in GitHub sponsors. While it is not yet possible for organizations to sponsor other accounts, I’m happy to report that our engineers are working to make that feature available. While we can’t yet say when our work will be done, we’ll be sure to get the word out as soon as we’ve made that feature available to the GitHub community.</p>
<p>In the meantime, many organizations have found a workaround in creating a user account for the specific purpose of sponsoring accounts and projects.</p>

**Comment by Piedone** [comment]
<p>I see, thanks Andrea!</p>

**Comment by ioquatix** [comment]
<p>I too am waiting for this feature as there are companies who want to sponsor me so I am leaving money on the table due to the lack of this feature. Is there any update regarding the time frame for beta/availability.</p>

**Comment by phase** [comment]
<p>We have recently run into this issue and are wondering if there’s an ETA for when this feature will be available.</p>

**Comment by oskarabc** [comment]
<p>Hello <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a> and colleagues: Our Company wants to change sponsorship from current channel to <a href="https://github.com/sponsors/mjackson" rel="nofollow noopener">https://github.com/sponsors/mjackson</a>, esp. because of GitHub matching contributions. Can’t seem to change ability to donate as the org I’m an admin/owner of.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for the ping <a class="mention" href="/oskarabc">@oskarabc</a> I think we’d be able to help you if you open a support request. If you haven’t already, please contact us at <a href="https://github.com/contact">https://github.com/contact</a> with the info so we can look into that further.</p>

**Comment by oskarabc** [comment]
<p>Thanks <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a>: I’ve created a ticket. Searching somewhere else there does seem an api endpoint to see if an “entity” (≠ org?) is eligible for sponsorship. Not sure if this applies to what I want to be able to do: <a href="https://docs.github.com/en/graphql/reference/unions#sponsor" rel="nofollow noopener">https://docs.github.com/en/graphql/reference/unions#sponsor</a></p>

**Comment by ArsalaBangash** [comment]
<p>Hi, I’ve run into this issue as well, and I was hoping to get an ETA. Thank you for your support!</p>

**Comment by RayCulp** [comment]
<p>We also need this feature. However, I am tentatively optimistic that I may have found a workaround. Even though I would technically be donating as an individual from Github’s point of view, it seems that I can enter the name and address of our company in my payment details and leave my own name out. This should, theoretically, result in some kind of a receipt of payment from Github that contains the company details but leaves me out of the picture. This should be acceptable to the tax authorities. Or does anyone sees a problem with this?</p>

**Comment by stancel** [comment]
<p>My organization is also interested in sponsoring a project for the framework we have used to build out application. Just adding a note here in hopes that I will be notified when this functionality is released so that we can then sponsor that project as our company/organization.</p>

**Comment by MRigal** [comment]
<p>We’d also be happy to use that feature soon too!</p>

**Comment by sandstrom** [comment]
<p>Any update on this (from Github staff)? For example, do you have any ETA to share with us?</p>

**Comment by Piedone** [comment]
<p>We’ve been sponsoring developers for a while, you can even see a new-ish “Sponsoring” section on the lower-right of our <a href="https://github.com/Lombiq" rel="noopener nofollow ugc">company profile</a>. We got into a preview program to try this, if it’s not yet publicly available then maybe if you ask <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a> nice enough you can join it too :).</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>:star: :rocket:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">
      <a href="https://github.blog/changelog/2020-12-08-github-sponsors-for-companies-now-available-in-beta/" target="_blank" rel="noopener">The GitHub Blog</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://github.blog/wp-content/uploads/2019/03/blog-card.png?fit=5001%2C2626" class="thumbnail" width="690" height="362"></div>

<h3><a href="https://github.blog/changelog/2020-12-08-github-sponsors-for-companies-now-available-in-beta/" target="_blank" rel="noopener">GitHub Sponsors for companies now available in beta - GitHub Changelog</a></h3>

<p>GitHub Sponsors for companies now available in beta</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by billhenn** [comment]
<p>Hello, I am set up as the “Billing Manager” for my organization, yet when I try to go sponsor an external developer using the my organization as the sponsor (instead of me personally), I get a “Help @[MyCompanyName] become a sponsor - Share this with a decision maker in the organization…” box and all the tier select buttons are disabled.</p>
<p>My question is, if I am a billing manager and can fully manage funding, why can’t I be the one who starts a sponsorship?  Seems like it should just work the same as if I was the organization admin, since I am a Billing Manager.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey there <a class="mention" href="/billhenn">@billhenn</a>, thank you for being here and my apologies for the confusion; currently,  it would have to be the organization’s admin who starts a sponsorship on behalf of their company. We are working on updating our documentation to reflect this guidance, and I’ve passed on your feedback to the team for consideration. I hope this helps, AG</p>

**Comment by billhenn** [comment]
<p>Thanks for the quick reply <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a>.  I think it makes sense that if a Billing Manager has control over the finances/budget/etc. that they should be able to set up the sponsors too.</p>
<p>One quick question, when you sponsor someone via your org (call it “MyOrg”), will it only say the MyOrg name in notification to the recipient and elsewhere on the site, or will it also mention the user who initiated the sponsorship on behalf of the org?  I do NOT want the latter, which is why I ask.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thank you for sharing additional context, I’ll share it with the team.</p>
<p>As far as the name in the notification, it would only be the former - so just MyOrg will be in notification to the recipient (the one being sponsored).</p>
<p>I hope this helps!<br>
AG</p>

**Comment by shaunak** [comment]
Hi @AndreaGriffiths11 , a bit late to the party, but do I have to be an org admin to sponsor as an org? Or can I just be a member?

**Reply by Piedone** [reply]
I don't know the exact permissions, but you have to definitely be more than a member.
